config/di.php

```diff
@@ -1,5 +1,19 @@
 <?php
 return [
+    'event_manager' => \DI\decorate(function($eventManager, $container) {
+        $template = $contaner->get('template');
+        $eventManager->attach('ERROR_DISPATCH', function (\GianArb\Penny\Event\HttpFlowEvent $event) use ($template) {
+            $e = $event->getException();
+
+            $response = $event->getResponse()->withStatus($e->getCode());
+            $event->setResponse($response);
+
+            $response->getBody()->write($template->render("error/40x", [
+                'title' => $e->getMessage(),
+                'exception' => $e,
+            ]));
```

Comment:
@gianarb returning nothing and returning `null` is the same thing in PHP. And sometimes you want to return null so that a service or parameter is set to `null`. So doing what you suggest would break that.